packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -96,9 +96,18 @@ module.exports = class AwsS3Multipart extends Plugin {
   createMultipartUpload (file) {
     this.assertHost()
 
+    let metadata = {}
+
+    Object.keys(file.meta).map(key => {
+      if (file.meta[key] !== null) {
```

Comment:
I think it'd be best if we also ignored `undefined` here, else the toString() line will crash:
```suggestion
      if (file.meta[key] != null) {
```